File: py_cpp/time.py
######################################
# This script is a way to verify
# the performance of python

# Author: @andvsilva 2022-12-12
######################################

import time
import sys

start_time = time.time()
number = 10**7 + 1

sum(range(number))

print(f">>> Python is {round(time.time() - start_time, 6)} in seconds.")
print("Sum is is =", sum(range(number)))

File: py_cpp/time.cpp
///////////////////////////////////////////////////
// This script is a way to verify the
// performance of C++

// Author: @andvsilva 2022-02-12
//////////////////////////////////////////////////

#include<stdio.h>
#include<stdint.h>
#include<inttypes.h>
#include<iostream>
#include<time.h>

using namespace std;

int main()
{ 
  // Time - Start
  clock_t start = clock();
  long int n = 10000000;
  int64_t sum = 0;
  double time_execution = 0;

  while(n>=0)
  {
    sum = sum + n;
    n--;
  } 
  clock_t end = clock();
  time_execution = end - start;
  printf("---> C is %f in seconds.\n", time_execution/CLOCKS_PER_SEC);
  printf("Sum is = %" PRIi64 "", sum);

  return 0;

}

